Norm/Database/Contexts/NormDbContext.cs

```diff
@@ -31,7 +22,7 @@ protected override void OnModelCreating(ModelBuilder builder)
         public DbContext Context => this;
 
         private readonly string dbConnectionString;
-        public ILoggerFactory loggerFactory;
+        public ILoggerFactory loggerFactory;        

```

Comment:
```suggestion
```

Reply:
If you are going to remove these from the constructor, should probably remove them as properties as well. 😂